### PR TITLE
[AIRFLOW-4765] Fix DataProcPigOperator execute method

### PR DESCRIPTION
**Pull request by nuclearpinguin**

### CODE REVIEW

File: tests/contrib/operators/test_dataproc_operator.py [hunk 1]
```diff
@@ -615,6 +616,32 @@ def test_dataproc_job_id_is_set():
             _assert_dataproc_job_id(mock_hook, dataproc_task)
 
 
+class DataProcPigOperatorTest(unittest.TestCase):
```
Comment (mik-laj) [inline tests/contrib/operators/test_dataproc_operator.py]: Can you also add system tests for this operator?

File: airflow/contrib/example_dags/example_gcp_dataproc_pig_operator.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+PROJECT_ID = os.environ.get('GCP_PROJECT_ID', 'an-id')
+REGION = os.environ.get('GCP_LOCATION', 'europe-west1')
+
+
+with models.DAG(
+    "example_gcp_dataproc_pig_operator",
+    default_args=default_args,
+    schedule_interval=None,
+) as dag:
+    create_task = DataprocClusterCreateOperator(
```
Comment (mik-laj) [inline airflow/contrib/example_dags/example_gcp_dataproc_pig_operator.py]: I think, you should keep variable name and task id in sync

**Comment by nuclearpinguin** [comment]
Should we merge also this one?

**Comment by potiuk** [comment]
Sure! Just open it against Apache repo :)

**Comment by nuclearpinguin** [comment]
Done, here is the PR:
https://github.com/apache/airflow/pull/5426

**Pull request closed without merging**